doc/devel/coding_style_guideline.rst

```diff
@@ -0,0 +1,169 @@
+=============================
+DWI_ML Coding Style Guideline
+=============================
+
+The main principles behind DWI_ML development are:
+
+* **Robustness**: the results of a piece of code must be verified
```

Comment:
Again, should you cite Dipy here for taking the same text? Is it plagiarism or is it fine?

Reply:
I added that to DIPY, probably inspired from many places, but yeah, why not.

Reply:
Done in ad7ee3c.